addons/batikvis/src/main/java/de/lmu/ifi/dbs/elki/visualization/gui/detail/DetailView.java

```diff
@@ -407,9 +407,7 @@ protected void redraw() {
         vis.incrementalRedraw();
       }
     }
-    if(active || true) {
-      refresh();
-    }
+    refresh();
```

Comment:
A FIXME or TODO would indeed be less confusing to show the intention of the `|| true`.